content/features/service_discovery.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "Service Discovery"
+file: "service_discovery"
+date: 2018-07-18T12:33:50+02:00
+draft: false
+youTubeID: oG-WDA-hmvA
+subTitle: "Services view focus on Istio observability"
+---
+* List of Services with Health status and error rate
```

Comment:
I am not sure what's the idea of @theute here, but perhaps we can just summarize in three lines the features.
In order to not need to click on the "more".

Perhaps in three lines:
- List of Services focus on Observability (Health, Error rate).
- Browse and validate Istio networking configuration per Service.
- Detailed Metrics and Tracing.

Or something like that, just an idea. I guess in the video it is showed more or less with a lot of details that perhaps don't need to be listed on the page but just to add some highlights.

Reply:
I don't know, maybe the Content will have a images and more description about how to do it things in next releases so I think that in the list of features we should make a resume or something like that and show all the information in the full page of the feature